src/routes/write-exercise/components/Feedback.tsx

```diff
@@ -56,6 +56,10 @@ const Feedback = ({ documentWithResult, submission, needsToBeRepeated }: Feedbac
     message = `${getLabels().exercises.write.feedback.almostCorrect1} „${submission}“ ${
       getLabels().exercises.write.feedback.almostCorrect2
     }`
+  } else if (!submission) {
```

Comment:
I think you could merge this additional `else if` with the else block since its exactly identical except the message. There you already have a condition where the else case is what you want. So you'd just have to update the condition there.

Reply:
ok, thank you